### PR TITLE
[内容修订] src/get-started/*

### PR DESCRIPTION
**Pull request by AlexV525**
### 解决的 Issues

cfug/team#19

### 更新内容描述

更新 `src/get-started` 所有文档。

### 关联上游

- https://github.com/flutter/website/pull/6937
- https://github.com/flutter/website/pull/6986
- https://github.com/flutter/website/pull/7004
- https://github.com/flutter/website/pull/7009
- https://github.com/flutter/website/pull/7042
- https://github.com/flutter/website/pull/7045

### CODE REVIEW

File: src/get-started/test-drive/_vscode.md [hunk 1]
```diff
@@ -2,17 +2,39 @@
...
+     打开 **View > Command Palette**。
+
   1. Enter a project name, such as `myapp`, and press **Enter**.
+
+     打开 **View > Command Palette**。
+
   1. Wait for project creation to complete and the `main.dart`
      file to appear.
 
+     打开 **View > Command Palette**。
```
Comment (Vadaski) [inline src/get-started/test-drive/_vscode.md:30]: 这里翻译是不是错位啦

**Review by Vadaski — COMMENTED** [review]
有一处需要修改的地方，辛苦 A 少再看看，其他我觉得 LGTM 啦

File: src/get-started/test-drive/_vscode.md (hunk 1 above)
Comment (Vadaski) [reply src/get-started/test-drive/_vscode.md:30]: 上面好几个都还没有翻译

**Review by Vadaski — APPROVED** [review]
LGTM

File: src/get-started/flutter-for/android-devs.md [hunk 2]
```diff
@@ -776,7 +808,12 @@ First, register the intent filter for all intents in `AndroidManifest.xml`:
 ```xml
 <activity
   android:name=".MainActivity"
-  android:launchMode="singleTop"
+  android:launch
+  
+  
+  
+  
+  Mode="singleTop"
```
Comment (AlexV525) [inline src/get-started/flutter-for/android-devs.md:816]:
```suggestion
  android:launchMode="singleTop"
```

**Pull request merged**